test/integration/router_test.go

```diff
@@ -526,6 +533,9 @@ func TestRouterPathSpecificity(t *testing.T) {
 	if err := waitForRoute(routeAddress, "www.example.com", "http", nil, tr.HelloPod); err != nil {
 		t.Fatalf("unexpected response: %q", err)
 	}
+	if err := waitForRoute(routeTestAddress, "www.example.com:80", "http", nil, tr.HelloPodPath); err != nil {
```

Comment:
my bad - its late ...  ignore my comments - I glossed over the `routeTestAddress` part.